src/metabase/api/pulse.clj

```diff
@@ -86,10 +86,10 @@
   [id]
   (let [card (Card id)]
     (read-check Database (:database (:dataset_query card)))
-    (let [data (:data (qp/dataset-query (:dataset_query card) {:executed_by *current-user-id*}))]
+    (let [result (qp/dataset-query (:dataset_query card) {:executed_by *current-user-id*})]
       {:status 200, :body (html [:html [:body {:style "margin: 0;"} (binding [render/*include-title* true
                                                                               render/*include-buttons* true]
-                                                                      (render/render-pulse-card card data))]])})))
+                                                                      (render/render-pulse-card card result))]])})))
```

Comment:
Yep that's the plan. Just wanted to check with you to make sure the implementation is solid first.